Lesson14/fercan/script.js

```diff
@@ -0,0 +1,28 @@
+// 9. Declare a variable number6 and initialize it to 6. Declare a variable number7 and initialize it to `++number6` (number6 incremented using prexif ++ operator). Log both values to the console.
+let number6 = 6 ;
+number7 = '++number6' ;
+   
+
+console.log('Exercise 9:, ${number7}');
+console.log('------------------------');
+
+
+// 10. Declare a variable number8 and initialize it to 8. Declare a variable number9 and initialize it to `number8++` (number8 incremented using postfix ++ operator). Log both values to the console.
+let number8 = 8;
+number9 = 'number8++';
+
+console.log('number8, number9');
+
+
+// 11. Declare a variable number10 and initialize it to -3. Use the postfix ++ operator to increment the value of num by 1, then multiply the result by 2 and log it to the console.
+let number10 = -3;
+number10 = (number10++)*2;
+console.log('number10');
```

Comment:
This console.log does not log the value of `number10` variable but only logs a string 'number10' (which is not really related to the `number10` variable).
```suggestion
console.log('Ex. 11: number10 is', number10);
```